php-cs-fixer

```diff
@@ -56,7 +56,7 @@ if (class_exists('Phar')) {
 
 if ($require) {
     // OK, it's not, let give Composer autoloader a try!
-    $possibleFiles = [__DIR__.'/../../autoload.php', __DIR__.'/../autoload.php', __DIR__.'/vendor/autoload.php'];
+    $possibleFiles = [__DIR__.'/../../autoload.php', __DIR__.'/../autoload.php', __DIR__.'/vendor/autoload.php', __DIR__.'/dev-tools/vendor/autoload.php'];
```

Comment:
it is a prod file. it must not be aware about `dev-tools`

Reply:
I agree that is the perfect solution.

Yet I can name [one file](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.0/src/Console/Command/HelpCommand.php#L287) that is aware of `dev-tools`.

Also, what harm can it make? Only if previous 3 paths are not matching that last one is even considered - and I don't believe would match for anyone.

In fact, only one test class was failing without this change - I can give it another try if you insist.

Reply:
Point existing code smell is not the best excuse to introduce another...

i consider referencing to dev file available in dev env only to be a code smell

Reply:
> i consider referencing in prod file to dev file available in dev env only to be a code smell

me too, but for me in this case it's a valid exception to do that

Reply:
on the second thought - and after cleaning the other smell - this needs fixing